src/drivers/spektrum_rc/spektrum_rc.cpp

```diff
@@ -50,8 +50,11 @@
 #include <uORB/topics/input_rc.h>
 
 // Snapdraogon: use J12 (next to J13, power module side)
+#if defined(__PX4_QURT)
+#define SPEKTRUM_UART_DEVICE_PATH "/dev/tty-1"
+#else
 #define SPEKTRUM_UART_DEVICE_PATH "/dev/tty-3"
```

Comment:
I think we can leave it for now. Something to think about later if we want to combine the RC input handling across more platforms.